website/src/content/docs/guides/vpn-setup.md

```diff
@@ -203,3 +203,35 @@ In Prowlarr, under `Settings -> Indexers -> Add [Indexer Proxies]`, select `Http
 If you added a `proxy` tag, make sure to also add that to the desired Indexers, under `Indexers -> Edit Indexer -> Tags`.
 
 Your indexer traffic will now be routed through the Gluetun HTTP proxy. Check the `qbittorrent-vpn` container logs to confirm.
+
+## Talos specific config
+
+In talos in order to use the tun interface for Glutun a workaround is needed otherwise you will get an error with permissions creating the tun interface.
```

Comment:
```suggestion
In talos in order to use the tun interface for Gluetun a workaround is needed otherwise you will get an error with permissions creating the tun interface.
```

Fixed typo